bin/packages/build.js

```diff
@@ -214,14 +206,14 @@ function buildPackage( packagePath ) {
 		nodir: true,
 	} );
 
+	// Build package CSS files
+	await buildPackageScss( packagePath );
+
 	process.stdout.write( `${ path.basename( packagePath ) }\n` );
 
 	// Build js files individually.
 	jsFiles.forEach( ( file ) => buildJsFile( file, true ) );
 
-	// Build package CSS files
```

Comment:
This is reordered to ensure the CLI logging looks/behaves the same as it did before.